phpstan.neon

```diff
@@ -19,4 +19,5 @@ parameters:
     ignoreErrors:
         # Put false positives here
         - '#PHPDoc tag @var for variable \$row contains unresolvable type#'
+        - "#Cannot access offset 'name' on array.*#"
```

Comment:
I would prefer to use this instead: https://phpstan.org/user-guide/ignoring-errors ... no need to ignore the error everywhere and np need to use a baseline imho

Reply:
my bad, didnt realize this option :), fixed